src/benefits/benefits_test.go

```diff
@@ -26,5 +27,7 @@ func TestGetBenefitsByIds(t *testing.T) {
 	if err != nil {
 		assert.Fail(t, "Error occured when getting benefits by id")
 	}
-	assert.Equal(t, benefitId, benefit.ID)
```

Comment:
benefitId can start as an int instead of converting

Reply:
Originally tried using ints throughout but this causes issues in the benefits.go handler, specifically when we try to check for the benefit based on the ID since you can't really check for an empty int in golang since the zero value is 0. Can use pointers or multiple returns to get around this but that is more work than the solution we ended up with